integration/src/integration/java/pl/allegro/tech/hermes/integration/HermesClientPublishingTest.java

```diff
@@ -67,27 +62,6 @@ public void shouldPublishUsingOkHttpClient() {
         runTestSuiteForHermesClient(client);
     }
 
-    @Test
-    public void shouldPublishUsingOkHttpClientUsingSSL() {
```

Comment:
I don't see any difference between this test and `shouldCommunicateWithHermesUsingHttp2`. I decided to remove it.